src/api/projects.js

```diff
@@ -0,0 +1,10 @@
+import { apiHandler } from './apiHandler';
+
+export const getProjectsApi = (searchTerm) => {
```

Comment:
no need for Api methods must describe what is happening so getProjects is enough

Reply:
done